src/input.css

```diff
@@ -1 +1,5 @@
 @import "tailwindcss";
+@import url('https://fonts.googleapis.com/css2?family=Outfit:wght@100..900&display=swap');
+@theme {
```

Comment:
can you add a space between `@import` and `@theme` for readability.

Reply:
You can click "Resolve conversation" to show that this comment has been adressed.